jsapp/js/components/permissions/userAssetPermsEditor.es6

```diff
@@ -416,38 +411,34 @@ class UserAssetPermsEditor extends React.Component {
             label={t('Edit Form')}
           />
 
-          <div className={classNames(
-            this.state.submissionsView === true ? 'user-permissions-editor__row user-permissions-editor__row--group' : ''
-          )}>
-            <Checkbox
-              checked={this.state.submissionsView}
-              disabled={this.state.submissionsViewDisabled}
-              onChange={this.onCheckboxChange.bind(this, 'submissionsView')}
-              label={t('View Submissions')}
-            />
+          <Checkbox
+            checked={this.state.submissionsView}
+            disabled={this.state.submissionsViewDisabled}
+            onChange={this.onCheckboxChange.bind(this, 'submissionsView')}
+            label={t('View Submissions')}
+          />
 
-            {this.state.submissionsView === true &&
-              <div>
-                <Checkbox
-                  checked={this.state.submissionsViewPartial}
-                  disabled={this.state.submissionsViewPartialDisabled}
-                  onChange={this.onCheckboxChange.bind(this, 'submissionsViewPartial')}
-                  label={t('Restrict to submissions made by certain users')}
+          {this.state.submissionsView === true &&
+            <div className='user-permissions-editor__sub-row'>
+              <Checkbox
+                checked={this.state.submissionsViewPartial}
+                disabled={this.state.submissionsViewPartialDisabled}
+                onChange={this.onCheckboxChange.bind(this, 'submissionsViewPartial')}
+                label={t('Restrict to submissions made by certain users')}
+              />
+
+              {this.state.submissionsViewPartial === true &&
+                <TagsInput
+                  value={this.state.submissionsViewPartialUsers}
+                  onChange={this.onSubmissionsViewPartialUsersChange}
+                  addOnBlur
+                  addKeys={[9, 13, 32]}
```

Comment:
Not a blocker at all, Just a thought I'd like to share. 
I could not remember that key code `9` was for, so I was wondering what would you think about using constants instead of these key codes? 
Like
**constants.es6**:
```
KEY_CODES = {
  ENTER: 13,
  SPACE: 32,
  TAB: 9
}
```

and using them in the code
`addKeys={[KEY_CODES.TAB, KEY_CODES.ENTER, KEY_CODES.SPACE]}`

Reply:
Good idea!